python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -179,14 +202,97 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
             {
               label: "Required resources",
               value:
-                Object.entries(actor.requiredResources).length > 0 &&
-                Object.entries(actor.requiredResources)
-                  .sort((a, b) => a[0].localeCompare(b[0]))
-                  .map(([key, value]) => `${value.toLocaleString()} ${key}`)
-                  .join(", ")
+                Object.entries(actor.requiredResources).length > 0
+                  ? Object.entries(actor.requiredResources)
+                      .sort((a, b) => a[0].localeCompare(b[0]))
+                      .map(([key, value]) => `${value.toLocaleString()} ${key}`)
+                      .join(", ")
+                  : null
             }
           ];
 
+    // Apply transformation to add styling for information field
+    const transforms: Record<
+      string,
+      (info: ActorInformation) => ActorInformation
```

Comment:
It would be better if this were explicitly allowed to be `undefined` since we're checking for that below.